variables.env

```diff
@@ -10,28 +10,28 @@ MEDIAWIKI_VERSION=1.40.1
 RELEASE_MAJOR_VERSION=1.40
 RELEASE_VERSION=1.40.1
 
-# WDQS_VERSION 0.3.135 is latest release on 2023-10-11
+# WDQS_VERSION 0.3.136 is latest release on 2023-11-09
 # as per https://github.com/wikimedia/wikidata-query-rdf/tags
 # and https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/
 WDQS_VERSION=0.3.135
 
-# Head of master on 2023-10-11
+# Head of master on 2023-11-09
 # as per https://github.com/wikimedia/wikidata-query-gui/
-WDQS_FRONTEND_COMMIT_HASH=4e94cdecec4aaa252465429dc9024b3dcc0056b5
+WDQS_FRONTEND_COMMIT_HASH=eb1825b3431f3adf47cd774bf52344024e2f11e9
 
-# Head of master on 2023-10-11
+# Head of master on 2023-11-09
 # as per https://github.com/magnusmanske/quickstatements/commits/master
 QUICKSTATEMENTS_COMMIT_HASH=c4b2c6b086b319aa32dcdd7a323edf188faaa66d
 
-# Head of master on 2023-10-11
+# Head of master on 2023-11-09
 # as per https://bitbucket.org/magnusmanske/magnustools/commits/branch/master
-MAGNUSTOOLS_COMMIT_HASH=11e19400271a47c4d7e927bcb7aa95f1bfaf38a9
+MAGNUSTOOLS_COMMIT_HASH=e4abdb6b3bedfdde0325df513b272e1972b1af36
 
-# Head of master on 2023-10-11
+# Head of master on 2023-11-09
 # as per https://github.com/ProfessionalWiki/WikibaseLocalMedia/tree/master
 WIKIBASELOCALMEDIA_COMMIT_HASH=6dcc6e82526896352b6bae1df11750f378201fcb
```

Comment:
this is surely right since it is the parent commit (https://github.com/ProfessionalWiki/WikibaseLocalMedia/commit/6dcc6e82526896352b6bae1df11750f378201fcb) to yours.

I would like to know though why we are not using the commit from the patch that you had to add on that code base, which is 74c3f70893ffd76feefa9f1e3d77052db20ec3ac?

Reply:
This has been changed, for some reason you did not see the change in your review. Did I forgot to push something? Not it is there https://github.com/wmde/wikibase-release-pipeline/pull/507/files#diff-3aa2b6a722522c6a53ed1ab4f2d690408c5fbf68597868a4e331fd13c4bc0933R30